### PR TITLE
Added Bernoulli distribution + tests

### PR DESCRIPTION
**Pull request by tdiethe**
*Issue #, if available:*
#21 

*Description of changes:*
Added Bernoulli distribution + tests

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: mxfusion/components/distributions/random_gen.py [hunk 1]
```diff
@@ -83,6 +91,19 @@ def sample_multinomial(data, get_prob=True, dtype='int32', F=None):
         return F.random.multinomial(
             data=data, get_prob=get_prob, dtype=dtype)
 
+    @staticmethod
+    def sample_bernoulli(prob_true=0.5, dtype='bool', F=None):
```
Comment (meissnereric) [inline mxfusion/components/distributions/random_gen.py]: Why do you pass in prob_true if you don't actually use it?

Comment (tdiethe) [reply mxfusion/components/distributions/random_gen.py]: that's a bug! good catch

**Comment by tdiethe** [comment]
> @tdiethe Could you just update your branch with the changes from the develop branch? This will trigger travis-ci to run.

done

File: mxfusion/components/distributions/random_gen.py [hunk 2]
```diff
@@ -83,6 +91,20 @@ def sample_multinomial(data, get_prob=True, dtype='int32', F=None):
...
+        Sample Bernoulli distributed variables
+
+        :param shape: Array shape of samples
+        :param prob_true: Probability of being true
+        :param dtype: data type
+        :param F: MXNet node
+        :return: Array of samples
+        """
+        F = get_default_MXNet_mode() if F is None else F
+        return F.random.uniform(low=0, high=1, shape=shape) > prob_true
```
Comment (meissnereric) [inline mxfusion/components/distributions/random_gen.py]: Do you need to pass dtype in anywhere here? Is it useful to have it as a parameter even if the outcome is a boolean array?

Comment (tdiethe) [reply mxfusion/components/distributions/random_gen.py]: Done, and added test

File: mxfusion/components/distributions/random_gen.py [hunk 3]
```diff
@@ -83,6 +91,20 @@ def sample_multinomial(data, get_prob=True, dtype='int32', F=None):
...
+        Sample Bernoulli distributed variables
+
+        :param shape: Array shape of samples
+        :param prob_true: Probability of being true
+        :param dtype: data type
+        :param F: MXNet node
+        :return: Array of samples
+        """
+        F = get_default_MXNet_mode() if F is None else F
+        return F.random.uniform(low=0, high=1, shape=shape, dtype=dtype) > prob_true
```
Comment (meissnereric) [inline mxfusion/components/distributions/random_gen.py]: Does this need to handle that shape handling we've discussed elsewhere?

Comment (tdiethe) [reply mxfusion/components/distributions/random_gen.py]: This is handling it correctly, since the parameters are both scalars.

**Pull request merged**